### DISCUSSION TITLE
Verification error of OpenFPGA

### DISCUSSION THREAD

**Discussion by AyaseErii** [lnis-uofu/OpenFPGA#410] [Q&A] [answered]
I started to learn the OpenFPGA recently. After the compile step, I launched a verification step but it shows that "ERROR (00_and2_MIN_ROUTE_CHAN_WIDTH) - Failed to execute openfpga flow - 00_and2_MIN_ROUTE_CHAN_WIDTH". I also put a screenshot below:
![d958aabaa1041beda03ec6289bffadd](https://user-images.githubusercontent.com/70917894/140388404-446987e6-a586-471f-89f6-dcf7148556e8.png)

I will appreciate it if someone can help me out with this issue. Thanks!

**Comment by tangxifan** [comment, marked as answer]
@AyaseErii It seems that you have a similar issue as #407 . My suggestion is to turn on ``--debug`` and ``--show_thread_logs`` options. Once the error message is clear. We will decide to patch or not.

**Reply by AyaseErii** [reply]
Thanks, Xifan! It turns out that I do not have a module named envyaml. Once I installed that module, the verification was successful.
